src/index.js

```diff
@@ -24,13 +24,25 @@ module.exports = {
       records = [];
     };
 
+    const close = async () => {
+      if (dbConnection && !config.dbConnection) {
```

Comment:
Can you add a test for this section please - that it does not close the provided connection